agent.go

```diff
@@ -1445,6 +1445,11 @@ func init() {
 		}
 		panic("--this line should have never been executed, congratulations--")
 	}
+
+	// agent won't receive anything from the stdin.
+	// Close stdin to avoid the agent receives signals
+	// (ctrl + c/d) from the stdin.
+	os.Stdin.Close()
```

Comment:
Good catch @devimc ! Could you tal at the rust agent to see if we need to do the same there please?

Reply:
@jodh-intel  sure thing

Reply:
@jodh-intel I couldn't reproduce it with the rust agent